IBM/carbon-addons-iot-react
# [HeaderAction] passing down of onClick handler

Closes #1150

**Summary**

Added missing onClick prop to HeaderGlobalAction

**Change List (commits, features, bugs, etc)**

- Added onClick prop to HeaderGlobalAction
- Added one test in Header to satisfy jest coverage

**Acceptance Test (how to verify the PR)**

1. Go to https://carbon-addons-iot-react.com/?path=/story/watson-iot-header--header-no-submenu
2. Open up the knobs panel and go to action tab
3. Click on action item button on right hand side
4. See no action being triggered
